share/dogecoin.conf

```diff
@@ -0,0 +1,463 @@
+##
+## dogecoin.conf configuration file.
+## Generated by contrib/devtools/gen-dogecoin-conf.sh.
+##
+## Lines beginning with # are comments.
+## All possible configuration options are provided. To use, copy this file
+## to your data directory (default or specified by -datadir), uncomment
+## options you would like to change, and save the file.
+##
+
+
+### Options
+
+
+# Print version and exit
+#version=1
+
+# Execute command when we see a really long fork (%s in cmd is replaced by
+# message)
+#alertnotify=<cmd>
+
+# Execute command when the best block changes (%s in cmd is replaced by
+# block hash, %i is replaced by block number)
+#blocknotify=<cmd>
+
+# If this block is in the chain assume that it and its ancestors are valid
+# and potentially skip their script verification (0 to verify all,
+# default:
+# e7d4577405223918491477db725a393bcfc349d8ee63b0a4fde23cbfbfd81dea,
+# testnet:
+# 199bea6a442310589cbb50a193a30b097c228bd5a0f21af21e4e53dd57c382d3)
+#assumevalid=<hex>
+
+# Specify directory where to write backups and data dumps (default
+# datadir/backups)
+#backupdir=<dir>
+
+# Specify configuration file (default: dogecoin.conf)
+#conf=<file>
```

Comment:
minor: file would be referencing itself? May want to create an exclusion for this

Reply:
It's the case for bitcoin.conf in fact. Indeed sounds weird but maybe if you have multiple configs you can avoid to specify the conf with putting one in default path and redirect from conf to the other. 
It can simplify the command maybe.

Reply:
Fair enough. I'll test whether that works separate from this PR - this is non-blocking for now.

Reply:
It's also on Bitcoin because to remove it we would need to create a specific regex or remove this line at the end of the execution. 
Don't know what can be better but it's not done on gen-bitcoin-conf.sh. 
Do you think that it's something I should try to do?

Reply:
I'd let it be for now. If this doesn't work we can also decide to change the behavior - I understand the use-case you describe for it. 

Consider this no problem for acceptance from my end, but let's keep it visible as something that we'd ideally investigate at some point before we release 1.15.0